pkg/storegateway/bucket.go

```diff
@@ -273,22 +272,20 @@ func (s *BucketStore) RemoveBlocksAndClose() error {
 
 // Stats returns statistics about the BucketStore instance.
 func (s *BucketStore) Stats(durations []time.Duration) BucketStoreStats {
```

Comment:
It looks like we don't actually use the map of block durations and counts for anything since https://github.com/grafana/mimir/pull/7309. Feel free to remove / change if you want (or we can do it in a follow-up PR).